### PR TITLE
EDSC-3835: Display the Harmony service name more prominantly on the access methods list

### PR DESCRIPTION
**Pull request by eudoroolivares2016**
# Overview

### What is the feature?

Moving the service name for `Harmony` customizable collection to the customize panel so user does not need to click on the `More Info` box. Rewriting tests that were using `Enzyme` to `react-testing-library`

### What is the Solution?

Small change to render the service name on the customizable option instead of under the `More Info` block

### What areas of the application does this impact?

List impacted areas.

# Testing

### Reproduction steps

- **Environment for testing:**
- **Collection to test with:**

1. Find a collection with `customizable` harmony service. Add the collection to your project go to the project page and hit customize

### Attachments
image

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist

- [x] I have added automated tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: static/src/js/components/AccessMethod/AccessMethod.js [hunk 1]
```diff
@@ -429,7 +430,7 @@ export class AccessMethod extends Component {
                 : (
                   <RadioList
                     defaultValue={selectedAccessMethod}
-                    onChange={(methodName) => this.handleAccessMethodSelection(methodName)}
+                    onChange={(methodName) => { this.handleAccessMethodSelection(methodName) }}
```
Comment (eudoroolivares2016) [inline static/src/js/components/AccessMethod/AccessMethod.js:433]: Will remove brackets

File: static/src/js/components/FormFields/AccessMethodRadio/__tests__/AccessMethodRadio.test.js [hunk 2]
```diff
@@ -1,153 +1,173 @@
...
 
   return {
-    enzymeWrapper,
-    props
+    onChange,
+    onClick
   }
 }
 
+// Setup DOM elements before each test
```
Comment (eudoroolivares2016) [inline static/src/js/components/FormFields/AccessMethodRadio/__tests__/AccessMethodRadio.test.js:36]: Remove debugging artifact

File: static/src/js/components/FormFields/AccessMethodRadio/__tests__/AccessMethodRadio.test.js [hunk 3]
```diff
@@ -1,153 +1,188 @@
...
   }
 }
 
 describe('AccessMethodRadio component', () => {
-  const { enzymeWrapper, props } = setup()
-
-  test('renders as a label', () => {
-    expect(enzymeWrapper.type()).toBe('label')
-  })
+  // TODO this is technically an implementation detail by react-test-library but, I could
```
Comment (eudoroolivares2016) [inline static/src/js/components/FormFields/AccessMethodRadio/__tests__/AccessMethodRadio.test.js:37]: Will remove after discussion

File: static/src/js/components/AccessMethod/__tests__/AccessMethod.test.js [hunk 4]
```diff
@@ -268,27 +238,28 @@ describe('AccessMethod component', () => {
...
-      const echoFormWrapper = customizationSection.find(ProjectPanelSection).at(1)
-      const suspenseComponent = echoFormWrapper.childAt(0)
-      const echoForm = suspenseComponent.childAt(0)
-
-      expect(echoForm.props().collectionId).toEqual(collectionId)
-      expect(echoForm.props().form).toEqual(form)
-      expect(echoForm.props().methodKey).toEqual('echoOrder0')
-      expect(echoForm.props().rawModel).toEqual(rawModel)
-      expect(typeof echoForm.props().onUpdateAccessMethod).toEqual('function')
+      screen.debug()
```
Comment (eudoroolivares2016) [inline static/src/js/components/AccessMethod/__tests__/AccessMethod.test.js:241]: Test props go into `@echo-forms` component

**Comment by trevorlang** [comment]
I think we may want to treat this differently in the UI. It seems like we might be able to make and improvement by incorporating it into the header of each radio item. Something like "Harmony (PODAAC Concise)" or "PODAAC Concise via Harmony". I will tag up with Alicia and see if she has a preference how we should handle it.

**Comment by eudoroolivares2016** [comment]
I see some merit to that @trevorlang lets tag up towmorrow

**Review by dpesall — APPROVED** [review]

**Review by trevorlang — APPROVED** [review]

**Pull request merged**